### ISSUE TITLE
Need an API to get the current caret rects

### ISSUE THREAD

**Issue by rniwa** [w3c/selection-api#58]
We need an API to return one, or two in the case of bidi, rects for the current caret position.

**Comment by caub** [comment]
it works on Firefox, (range.getBoundingClientRect) and should work soon on chrome when they fix it for collapsed ranges

**Comment by yosinch** [comment]
Isn't CaretPosition.getClientRect() in [1] enough?

BTW, Is bounding box enough for application? It is better to return DOMQuad[2] in CSSOM View API, then we author use DOMQuad.bounds if they want to use bounding rectangle.

[1] https://drafts.csswg.org/cssom-view/
[2] https://www.w3.org/TR/geometry-1/#DOMQuad

**Comment by rniwa** [comment]
No, `CaretPosition.getClientRect()` would return one rect but caret could appear at two different locations simultaneously in some browsers.

**Comment by johanneswilm** [comment]
Yes, or the other way round, at least that's how it currently is.
